src/test/java/core/basesyntax/service/impl/FileReaderImplTest.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.service.FileReader;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.Test;
+
+public class FileReaderImplTest {
+    private static final FileReader reader = new FileReaderImpl();
+    private static final String EXISTING_FILE_PATH = "src/test/resources/inputTest.csv";
+    private static final String NON_EXISTING_FILE_PATH = "src/test/resources/noSuchFile.csv";
+    private static final String EMPTY_FILE_PATH = "src/test/resources/empty.csv";
+
+    @Test
+    public void readFromExistingFile_Ok() {
+        List<String> expected = List.of("type,fruit,quantity", "b,banana,20", "s,apple,15");
+        assertEquals(expected, reader.readFromFile(EXISTING_FILE_PATH));
+    }
+
+    @Test(expected = RuntimeException.class)
+    public void readFromNonExistingFile_notOk() {
+        reader.readFromFile(NON_EXISTING_FILE_PATH);
+    }
+
+    @Test
+    public void readFromEmptyFile() {
+        List<String> expected = new ArrayList<>();
+        List<String> actual = reader.readFromFile(EMPTY_FILE_PATH);
+        assertEquals(expected, actual);
```

Comment:
assertEquals(0, emptyList.size())

Reply:
Why not? The result will mean the same. Empty list or list.size()==0 means that list read from empty file is empty